### PR TITLE
Export enums and related types like Prisma does

### PR DESCRIPTION
**Pull request by jvandenaardweg**
I noticed the PR https://github.com/valtyr/prisma-kysely/pull/29 fixing issue https://github.com/valtyr/prisma-kysely/issues/27 wasn't really doing it for me. Back then didn't had the time to look into fixing it, so I ignored it and kept using the types from the Prisma client.

With the current export, the enum values are seen as a `string` instead of their value, like `"NL"`, even in a zod schema like:

```typescript
import { Language } from './types';

const currentLanguage = Language.NL // string

const schema = z.object({
  language: z.nativeEnum(Language)
  // language = { NL: string, EN: string }
});

export type SchemaLanguage = z.infer<typeof schema>['language'];

// type SchemaLanguage = string
 ```

This PR fixes that by exporting the enum's [exactly](https://github.com/prisma/prisma/blob/ccf3df016e80fe72ba567f54a9c669ce2b926c9a/packages/client/src/generation/TSClient/Enum.ts#L40) like Prisma does it, requiring no additional changes to code that uses the enum types and values from the Prisma client. So making the switch to Kysely even easier.

So, in summary, this PR generates:

```typescript
export const Language = {
  NL: "NL",
  EN: "EN",
};
export type Language = (typeof Language)[keyof typeof Language];
```

instead of:
```typescript
export const Language = {
  NL: "NL",
  EN: "EN",
};
export type Language = 'NL' | 'EN';
```

So we get this working correctly:

```typescript
import { Language } from './types';

const currentLanguage = Language.NL // "NL"

const schema = z.object({
  language: z.nativeEnum(Language)
  // language = { NL: "NL", EN: "EN"}
});

export type SchemaLanguage = z.infer<typeof schema>['language'];

// type SchemaLanguage = "NL" | "EN"
 ```

### CODE REVIEW

**Review by valtyr — APPROVED** [review]
Hah I guess I should have tried using the native enums before I released them. Thank you for catching this and submitting a PR.

**Comment by jvandenaardweg** [comment]
Could be... Also noticed I haven't tested them well when validation your PR when you asked me :-)

fyi: the PR about the enum map https://github.com/valtyr/prisma-kysely/pull/39 kinda conflicts with this one, as the type changed, the test will probably fail when this current PR is merged. Happy to resolve the issue once you merge things.

**Comment by valtyr** [comment]
@jvandenaardweg I made a small change by adding `as const` to the enum declarations. Thought we might as well use this PR to clean things up 😄. Thank you again for your contribution! I'll merge this in now.
